### PR TITLE
fix(tree): the symbols of image type do not display at the first rendering. close #12279.

### LINKED ISSUE #11860: 树图使用图片symbol后，节点只有点击才会展示
### Version
4.5.0

### Steps to reproduce
打开官网树图demo https://echarts.apache.org/examples/zh/editor.html?c=tree-basic
替换内容为
```javacript
myChart.showLoading();
const url = `data:image/svg xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="0px" y="0px" viewBox="0 0 100 100" enable-background="new 0 0 80 40" xml:space="preserve">
    <rect x="0" y="0" width="2px" height="100%" fill="%232DAFF9" />
    <rect  x="3" y="0" width="70px" height="100%" fill="%232DAFF9" />
</svg>`

$.get(ROOT_PATH + 'data/asset/data/flare.json', function (data) {
    myChart.hideLoading();

    echarts.util.each(data.children, function (datum, index) {
        index % 2 === 0 && (datum.collapsed = true);
    });

    const d = {
     "name": "flare",
     "value": "70人，80%",
     "children": [
      {
       "name": "analytics",
       "value": "70人，80%",
       "children": [
        {
         "name": "cluster",
         "children": [
          {"name": "AgglomerativeCluster", "value": 3938},
          {"name": "CommunityStructure", "value": 3812},
          {"name": "HierarchicalCluster", "value": 6714},
          {"name": "MergeEdge", "value": 743}
         ]
        },
        {
         "name": "graph",
         "children": [
          {"name": "BetweennessCentrality", "value": 3534},
          {"name": "LinkDistance", "value": 5731},
          {"name": "MaxFlowMinCut", "value": 7840},
          {"name": "ShortestPaths", "value": 5914},
          {"name": "SpanningTree", "value": 3416}
         ]
        },
        {
         "name": "optimization",
         "children": [
          {"name": "AspectRatioBanker", "value": 7074}
         ]
        }
       ]
      }]
    };

    myChart.setOption(option = {
        tooltip: {
            trigger: 'item',
            triggerOn: 'mousemove'
        },
        series: [
            {
                type: 'tree',

                data: [d],

                top: '1%',
                left: '7%',
                bottom: '1%',
                right: '20%',
                symbol: `image://${url}`,
                symbolSize: [80, 40],
                itemStyle: {
                    color: '#2DAFF9',
                    borderWidth: 0,
                },
                label: {
                    normal: {
                        position: ['50%', '50%'],
                        verticalAlign: 'middle',
                        align: 'left',
                        fontSize: 9,
                        formatter: function(params) {
                            return `{name|${params.name}}\n${params.value}`
                        },
                        rich: {
                            name: {
                                color: '#fff',
                            },
                            
                        }
                    },
                    
                },

                expandAndCollapse: true,
                animationDuration: 550,
                animationDurationUpdate: 750
            }
        ]
    });
});
```

### What is expected?
节点背景图片默认展示

### What is actually happening?
节点背景图片初始不展示，点击后才会展示

### LINKED ISSUE #12279: symbol 设置图片以后  加载树形图 点击节点后才会加载图片
刷新页面,都不会显示自定义的图片,点击节点后才会显示

### PR DESCRIPTION
**Pull request by plainheart**
## Brief Information

This pull request is in the type of:

- [x] bug fixing
- [ ] new feature
- [ ] others

### What does this PR do?

Fixes #12279
Fixes #11860

### Fixed issues

- #12279
- #11860

### Before: What was the problem?
The symbols of image type do not display at the first rendering. Please refer to #12279, #11860 for more details.

Incorrect View

### After: How is it fixed in this PR?

Correct View

After creating the instance of `SymbolClz`, update its data immediately but not wait until the next update rendering if the type of symbol is image.

### Related test cases

Refer to `test/tree-image2.html` for a test case.

### CODE REVIEW

File: src/chart/tree/TreeView.js [hunk 1]
```diff
@@ -379,9 +379,8 @@ function updateNode(data, dataIndex, symbolEl, group, seriesModel, seriesScope)
         // Fix #12279.
         // if the type of symbol is image,
         // update symbol's data immediately but not wait until the next update rendering.
-        var symbolType = symbolEl._symbolType;
-        symbolType.indexOf('image://') === 0 
-            && symbolEl.updateData(data, dataIndex, seriesScope);
+        var symbolType = symbolEl.getSymbolPath().type;
+        symbolType === 'image' && symbolEl.updateData(data, dataIndex, seriesScope);
```
Comment (plainheart) [inline src/chart/tree/TreeView.js:383]: I'm thinking which scheme will be better here?

Before
```js
var symbolType = symbolEl._symbolType; // easy to get, but it is private, can I access it here?
symbolType.indexOf('image://') === 0 && symbolEl.updateData(data, dataIndex, seriesScope);
```
Now
```js
var symbolType = symbolEl.getSymbolPath().type;
symbolType === 'image' && symbolEl.updateData(data, dataIndex, seriesScope);
```

Comment (100pah) [reply src/chart/tree/TreeView.js:383]: @plainheart 
`updateData` has been called in the constructor of `Symbol`, I think it should better not be called again here.
The actual reason of this issue seams to be in: 
https://github.com/apache/incubator-echarts/blob/4.8.0/src/chart/helper/Symbol.js#L241
The reset of "image" symbol should not set `opacity` to `null`.
`null` is an invalid value for `opacity`.
I think here the opacity should be reset as `1`. 
And that can fix this issue.

Comment (plainheart) [reply src/chart/tree/TreeView.js]: Yes, you are totally right. I've made a new commit for this. https://github.com/apache/incubator-echarts/pull/12367/commits/48d54d8b604418b97b74e4da688c6a9699084d61

**Comment by pissang** [comment]
Looks good

**Pull request merged**